web-components/src/utils/styles/cn.ts

```diff
@@ -0,0 +1,6 @@
+import { ClassValue, clsx } from 'clsx';
+import { twMerge } from 'tailwind-merge';
+
+export function cn(...inputs: ClassValue[]) {
```

Comment:
@flagrede I'm not actually sure what this is for and when I should use it. Can you add some documentation?

Reply:
Added some documentation here: [52ce3b7](https://github.com/regen-network/regen-web/pull/2167/commits/52ce3b76d43379c1d034d064e7c229dab80ad0b1)

It allows to override class names. For example, if you have this:
`const Button => (className) => <button className={cn('text-primary', className)}`
then calling
`<Button className="text-secondary" />`
will override text-primary by text-secondary